test/node_modules/mapbox-gl-js-test.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
```

Comment:
So it can be required as `require('mapbox-gl-js-test')`, without a relative path which requires updating if the test file is moved up or down the directory hierarchy.

I'd like to do something similar for library requires, replacing `require('../../../js/foo/bar')` with `require('js/foo/bar')` via [app-module-path](https://www.npmjs.com/package/app-module-path).